Formula/srcml.rb

```diff
@@ -0,0 +1,223 @@
+class Srcml < Formula
+  desc "convert source code to XML"
+  homepage "http://www.srcml.org/"
+  url "http://131.123.42.38/lmcrs/beta/srcML-src.tar.gz"
+  version "0.9.5"
+  sha256 "55dd2115548e270724af4251187343656d2dfda0e7d372fee15ae27262e3fa8e"
+
+  depends_on "cmake" => :build
+  depends_on "LibArchive" => :build
+  depends_on "antlr@2" => :build
+  depends_on "libantlr3c" => :build
+  depends_on "Boost" => :build
+  depends_on "libxml2"
+
+  def install
+    system "cmake", "-G", "Unix Makefiles", "-DCMAKE_C_FLAGS_RELEASE=-DNDEBUG", "-DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG", "-DCMAKE_INSTALL_PREFIX=#{prefix}", "-DCMAKE_BUILD_TYPE=Release", "-DCMAKE_FIND_FRAMEWORK=LAST", "-DCMAKE_VERBOSE_MAKEFILE=ON", "-Wno-dev"
+    system "make", "install"
+  end
+
+  patch :DATA
```

Comment:
Most of the fixes are for getting cmake to compile and upload the built target to the Cellar folder rather than the /usr/local/lib folder (somehow there is a permission error which I couldn't figure out why). The other fix is with respect to an error reported by the new version of clang compiler.